sdk/cosmos/azure-cosmos/azure/cosmos/aio/cosmos_client.py

```diff
@@ -261,12 +286,11 @@ async def create_database_if_not_exists(  # pylint: disable=redefined-builtin
                 **kwargs
             )
 
-    def get_database_client(self, database):
-        # type: (Union[str, DatabaseProxy, Dict[str, Any]]) -> DatabaseProxy
+    def get_database_client(self, database: Union[str, DatabaseProxy, Dict[str, Any]]) -> DatabaseProxy:
         """Retrieve an existing database with the ID (name) `id`.
 
         :param database: The ID (name) representing the properties of the database to read.
-        :type database: str or dict(str, str) or ~azure.cosmos.DatabaseProxy
+        :type database: str or Dict[str, Any] or ~azure.cosmos.DatabaseProxy
```

Comment:
The description here seems inaccurate. It sounds like it is either the name or the actual properties themselves (as a dict or object).